releases_test.go

```diff
@@ -148,6 +148,11 @@ func findReleases(provider string, archived bool) ([]v1alpha1.Release, error) {
 		if err != nil {
 			return nil, microerror.Mask(err)
 		}
+		if strings.Contains(release.Name, "azure") || strings.Contains(release.Name, "capa") {
```

Comment:
CAPA and CAPZ (Azure) are now using a different release name "aws-25.0.0", "azure-25.0.0". In order to satisfy the tests I slightly changed it so it can be verified.